include/boost/multiprecision/cpp_dec_float.hpp

```diff
@@ -69,6 +71,11 @@ template <unsigned Digits10, class ExponentType, class Allocator>
 class cpp_dec_float
 {
  private:
+   static BOOST_CONSTEXPR_OR_CONST boost::uint32_t pow10_maker(boost::uint32_t n)
```

Comment:
Since from the next release onwards we're moving to C++11 or later, this could just be straight `constepxr` and we can drop the macros now?